mobile/mobile.go

```diff
@@ -150,6 +150,27 @@ func (w *Wrapper) IsSignedIn() bool {
 	return si
 }
 
+// SignOut calls core SignOut
+func (w *Wrapper) UpdateThread(mnemonic string, name string) error {
```

Comment:
@sanderpick if i just push this in here, that isolates this change a lot. pretty simple.

Reply:
okay sweet